capstone/src/main/java/com/google/sps/servlets/StudySetServlet.java

```diff
@@ -175,6 +189,131 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     String requestResultJSON = gson.toJson(getRequestResult(request));
     response.getWriter().println(requestResultJSON);
   }
-  //TODO doPost for creating and storing a study set
 
+  public void doPost(HttpServletRequest request, HttpServletResponse response)
+    throws IOException {
+    HashMap<String, Object> result_par = new Gson()
```

Comment:
Not sure what the `par` is supposed to stand for, guessing parameters but I'd use params if so. Also needs to be camel case: https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names